app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml

```diff
@@ -18,7 +18,7 @@
       data-mage-init='{
           "multiShipping":{},
           "cartUpdate": {
-               "validationURL": "/multishipping/checkout/checkItems",
+               "validationURL": "<?= /* @escapeNotVerified */ $block->getUrl('multishipping/checkout/checkItems') ?>",
```

Comment:
Please, avoid usage `escapeNotVerified` in the new or changed code, it's better to escape current string

Reply:
Hi @sidolov 
I have updated the change, please review it.

Reply:
Perfect! Thanks!